python/adelphi/adelphi/nb.py

```diff
@@ -132,4 +132,4 @@ def __build_schema(self):
         main_write_block = {"name":"main-write", "tags":{"phase":"main", "type":"write"}, "params":cl_ratio_map, "statements": [self.__build_main_write_statement()]}
         root["blocks"] = [rampup_block, main_read_block, main_write_block]
 
-        return yaml.dump(root, default_flow_style=False)
+        return yaml.dump(root, default_flow_style=False).strip()
```

Comment:
I was seeing strange diff failures due to added or missing blank lines in diff output.  Underlying cause appeared to be differences in writing to stdout vs. whatever the Python YAML package does here.  Adding this strip() call seemed to bring everybody into agreement on the minimal case (i.e. the fewest number of blank lines possible).